### ISSUE TITLE
Migration rollback doesn't work

### ISSUE THREAD

**Issue by evan-mark4s** [phalcon/phalcon-devtools#520]
Test case: I have a migration version 1.0.2 which contains 1 file: accounts.php, then I have migration version 1.0.3 which contains 1 file: admins.php with functions up() and down(), no morph().

I try migrating back from 1.0.3 to 1.0.2 using `phalcon migration run --version=1.0.2` but the down() function is not called. I tried inserting a small 
```
echo "$fromVersion -> $toVersion ($tableName)\n";
```
on function migrate() in Mvc\Model\Migrations.php and this is what I got
```
1.0.3 -> 1.0.3 (admins)
1.0.3 -> 1.0.2 (accounts)
```
So it seems the rollback for `admins` table was never called. Please check.

**Comment by sergeyklay** [comment]
What kind of rollback you are mean? Could you please provide migrations to reproduce?

**Comment by joel-depiltech** [comment]
Up !
I have the same issue. Do you have a solution for me ?

**Comment by sergeyklay** [comment]
Can you please provide script to reproduce?

**Comment by joel-depiltech** [comment]
This is my migration file : 
<?php 

use Phalcon\Db\Column;
use Phalcon\Db\Index;
use Phalcon\Db\Reference;
use Phalcon\Mvc\Model\Migration;

class CiUsersMigration_101 extends Migration
{
    public function up()
    {
       self::$_connection->update(
            "ci_users",
            array("username"),
            array("Astro Boy"),
            "id = 2"
        );
    }

    public function down()
    {
        self::$_connection->update(
            "ci_users",
            array("username"),
            array("Paul"),
            "id = 2"
        );
    }
}

When I upgrade to version 1.0.1, the username of id=2 changed to "Astro Boy" , but when I downgrade to  version 1.0.0 the username never changed to "Paul"

**Comment by joel-depiltech** [comment]
Up ! Please do you have an issue for me ?

**Comment by sergeyklay** [comment]
I'll try to deal with this in the near future

**Comment by sergeyklay** [comment]
@evan-mark4s @joel-depiltech Can you please check `2.0.x` branch?

**Comment by joel-depiltech** [comment]
Thanks. I will check on next Tuesday. I will come back to you...

**Comment by sergeyklay** [comment]
@joel-depiltech

**Comment by sergeyklay** [comment]
I think issue is fixed `2.0.x` branch. This will be opened if the issue is actual
